server/chat.ts

```diff
@@ -1847,15 +1847,23 @@ export const Chat = new class {
 
 	packageData: AnyObject = {};
 
+	getPluginName(file: string) {
+		const ext = pathModule.extname(file);
```

Comment:
`lastIndexOf('.')` was working fine before... did we really need to mess up our `path` import for this?

Reply:
Reverted.